StudentsDB/src/test/java/com/musala/simple/students/db/MongoDBCopy.java

```diff
@@ -0,0 +1,45 @@
+package com.musala.simple.students.db;
+
+import java.net.InetSocketAddress;
+
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.MongoClient;
+import com.mongodb.ServerAddress;
+import com.musala.simple.students.db.impl.MongoDB;
+import com.musala.simple.students.db.mappers.MongoObjectMapper;
+
+import de.bwaldvogel.mongo.MongoServer;
+import de.bwaldvogel.mongo.backend.memory.MemoryBackend;
+
+public class MongoDBCopy extends MongoDB {
```

Comment:
Better name this TestMongoDB.

Reply:
I've renamed it to MongoDBForTest and moved it as an inner clas for the MongoDBTest class as it is used for testing purposess.